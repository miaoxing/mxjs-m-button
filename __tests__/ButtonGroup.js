import React from 'react';
import {ButtonGroup} from '..';
import {Button, ButtonGroup} from '..';
import {render} from '@testing-library/react';

describe('ButtonGroup', () => {
  test('basic', () => {
    const {container} = render(<ButtonGroup>
      <Button>foo</Button>
      <Button>bar</Button>
    </ButtonGroup>);
    expect(container).toMatchSnapshot();
  });
});
