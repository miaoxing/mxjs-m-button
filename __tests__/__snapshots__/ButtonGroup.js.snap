// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonGroup basic 1`] = `
<div>
  <taro-view-core
    class="mx-btn-group"
  >
    <taro-button-core
      class="mx-btn"
    >
      foo
    </taro-button-core>
    <taro-button-core
      class="mx-btn"
    >
      bar
    </taro-button-core>
  </taro-view-core>
</div>
`;
